ui/src/views/admin/settings/SettingsView.vue

```diff
@@ -1,82 +1,145 @@
 <template>
   <v-container
     id="admin-setting-view"
-    fluid
     tag="section"
   >
     <v-row justify="center">
-      <v-col
-        cols="8"
-        md="8"
-      >
-        <material-card
-          color="primary"
+      <v-col :cols="isMobile? 12 : 8">
```

Comment:
The card size is bigger, should be 10~12. I changed it to 12, but feel free to adjust should you feel that 10 is OK.